facebook-auth-playground/frontend/index.js

```diff
@@ -0,0 +1,53 @@
+function onFacebookSDKLoaded() {
```

Comment:
It's 99% of the work of creating a library for FB login for us (just export the login/logout function, replace some stuff like console.log with nice interface to get events and replace some interface considerations like replacing callbacks with Promises). Doesn't it worth to change to library as part of this PR?

Reply:
we won't use any of this code,
we'll need to install this types package https://www.npmjs.com/package/@types/facebook-js-sdk
and call the same functions i used here